audio.c

```diff
@@ -43,6 +44,26 @@ bool isPlaying() {
   return ma_sound_is_playing(&sound);
 }
 
+void nextSong() {
+  if (currSong == NULL || currSong->next == NULL)
+    return;
+  currSong = currSong->next;
+  fprintf(log_file, "currSong = %s, currSong->prev = %s\n", currSong->title, currSong->prev->title);
+  ma_sound_uninit(&sound);
+  ma_sound_init_from_file(&engine, currSong->path, 0, NULL, NULL, &sound);
+  ma_sound_start(&sound);
+}
+
+void prevSong() {
```

Comment:
```suggestion
void prevSong() {
```

`void playPrevSong() {`

potential ambiguity